docs/index.md

```diff
@@ -9,16 +9,14 @@
 
 ROCm is an open-source software platform optimized to extract HPC and AI workload
 performance from AMD Instinct accelerators and AMD Radeon GPUs while maintaining
-compatibility with industry software frameworks. For more information, see [What is ROCm?](./what-is-rocm.rst)
+compatibility with industry software frameworks. For more information, see
+[What is ROCm?](./what-is-rocm.rst)
 
-If you're using Radeon GPUs, consider reviewing {doc}`Radeon-specific ROCm documentation<radeon:index>`.
-
-Installation instructions are available from:
+ROCm supports multiple programming
+languages such as {doc}`HIP (Heterogeneous-Compute Interface for Portability)<hip:index>`, OpenCL,
+and OpenMP, as explained in the [Programming guide](./how-to/hip_programming_guide.rst).
```

Comment:
OpenMP is not mentioned on the programming guide's page, and, to be pedantic again, is not considered to be a programming language.

Reply:
Updated.